### PR TITLE
Saiyam Name Card

### PR DESCRIPTION
**Pull request by SaiRev0**

### CODE REVIEW

File: utils/contributors.js [hunk 1]
```diff
@@ -5,8 +5,32 @@ const CONTRIBUTERS = [
         image: "/images/cops.jpg",
         description: "Hello everyone, this is the first card. Go ahead, add your own!",
         github: "https://github.com/COPS-IITBHU",
+    },{
+        handle_name: 'SaiRev0',
+        name: "Saiyam Jain",
+        image: "/images/saiyam.png",
```
Comment (debarati-06) [inline utils/contributors.js:11]: You have not added any image file. If you prefer to to add any images then remove the image path line from code.

**Review by debarati-06 — CHANGES_REQUESTED** [review]

**Comment by SaiRev0** [comment]
but i have uploaded a image also

**Comment by SaiRev0** [comment]
is it correct now??

**Review by debarati-06 — APPROVED** [review]
LGTM! Task completed 🎉

**Comment by debarati-06** [comment]
> but i have uploaded a image also

Earlier the image file was missing in added files

**Comment by debarati-06** [comment]
> is it correct now??

Yes it looks perfect now !

**Pull request closed without merging**